docs/building/macos.md

```diff
@@ -3,7 +3,7 @@
 This guide supplements the [Linux instructions](./linux.md), as
 building on macOS is almost identical.
 
-.NET Core 2.0 (and by transitivity, us) only supports macOS 10.12+.
+.NET Core 2.x (and by transitivity, us) only supports macOS 10.12+.
```

Comment:
Does the third bullet item also need to be changed?  

Downloads and installs a preview version of .NET Core SDK 2.0 to ~/.dotnet